### ISSUE TITLE
q.sanityImage returns as any

### ISSUE THREAD

**Issue by JohnGemstone** [FormidableLabs/groqd#109]
Hey!

Seems like since v0.12.0 `q.sanityImage` returns `any`. 

Reproduction below, just hover over the types to see how they evaluate:
https://stackblitz.com/edit/react-ts-z3aiuj?file=App.tsx

```tsx
import { z } from 'zod';
import { q } from 'groqd';

const query = q('*').grab({ image: q.sanityImage('image') });

// from groqd@0.12.0 image type is 'any'
type Foo = z.infer<typeof query.schema>;
```

**Comment by gksander** [comment]
Interesting, I'm seeing this as well. Let me take a peak and see what's going on here.

**Comment by gksander** [comment]
Hmm, I've tracked it down to [this change](https://github.com/FormidableLabs/groqd/pull/98/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80). Not sure yet why that overload is causing this, but I've got a place to investigate.

**Comment by gksander** [comment]
Some weird TypeScript stuff going on here, but TL;DR is to import `sanityImage` directly from `groqd` (like `import { sanityImage } from "groqd";`) instead of using `q.sanityImage`. This should be available in `groqd^0.1.4.0`, and should resolve the type issue you're seeing here. Thank you for reporting this! If this, for some reason, doesn't resolve your issue – ping back here so we can reopen this.

**Comment by JohnGemstone** [comment]
Awesome! Importing looks good, thanks again
